code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -43,16 +43,6 @@
 	src.add_fingerprint(usr)
 	return
 
-/obj/item/weapon/card/data/clown

```

Comment:
Except we have away missions and lavaland templates, if you are going to use it, just readd it. Besides, its shouldn't be a child of ID cards.